dev-client/src/screens/ProjectTeamScreen/components/UserItem.tsx

```diff
@@ -88,7 +88,7 @@ export const UserItem = ({
     <MenuItem
       icon={<ProfilePic user={user} />}
       label={userLabel}
-      pill={<RolePill membership={membership} />}
+      pill={<RoleChip membership={membership} />}
```

Comment:
While we're here, can we rename this property to "chip"? Another thing to refer to Courtney wrt. updating the documented property in `MenuItem`.